pyproject.toml

```diff
@@ -30,6 +30,8 @@ pyusb = "^1.2.1"
 tm-devices = "^1.1.0"
 watchfiles = "^0.21.0"
 keyrings-cryptfile = "^1.3.9"
+pytest = "^7.4.4"
```

Comment:
should be in dev dependency instead, remove those 2 and then `poetry add pytest -G dev`

Reply:
The test sequencer is calling pytest, so it need to be packaged in now

Reply:
Aha true! Forget abt this one 👍